test/extended/builds/s2i_quota.go

```diff
@@ -55,7 +55,8 @@ var _ = g.Describe("[Feature:Builds][Conformance] s2i build with a quota", func(
 				g.By("expecting the build logs to contain the correct cgroups values")
 				buildLog, err := br.LogsNoTimestamp()
```

Comment:
```suggestion
				//buildLog, err := br.LogsNoTimestamp()
				_, err := br.LogsNoTimestamp()
```

Reply:
or you can comment out the log fetch entirely since there isn't much point in doing it if we're not going to look at the log content.

Reply:
Done, thanks!